vpc/vpc-2azs.yaml

```diff
@@ -220,6 +220,9 @@ Outputs:
   TemplateID:
     Description: 'cloudonaut.io template id'
     Value: 'vpc/vpc-2azs'
+  StackName:
+    Description: 'Nested stack name'
```

Comment:
could you change 'Nested stack name' to 'Stack name' only? It's not really nested until you put it into another stack.
Same applies to the other templates